### PR TITLE
Some miscellaneous improvements

### PR DESCRIPTION
**Pull request by allinkdev**

### CODE REVIEW

**Comment by VideoGameSmash12** [comment]
Could you like, use Allman's style for the sake of code consistency?

**Comment by allinkdev** [comment]
> Could you like, use Allman's style for the sake of code consistency?

Tried to, but I know that I messed up in a few places. Will commit the required formatting changes shortly.

**Comment by VideoGameSmash12** [comment]
Flags do not work properly
```none
com.google.gson.JsonIOException: Failed making field 'java.util.regex.Pattern#pattern' accessible; either change its visibility or write a custom TypeAdapter for its declaring type
        at com.google.gson.internal.reflect.ReflectionHelper.makeAccessible(ReflectionHelper.java:22)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:158)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
        at com.google.gson.Gson.getAdapter(Gson.java:501)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:116)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:165)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
        at com.google.gson.Gson.getAdapter(Gson.java:501)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:116)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:165)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
        at com.google.gson.Gson.getAdapter(Gson.java:501)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:116)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:165)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
        at com.google.gson.Gson.getAdapter(Gson.java:501)
        at com.google.gson.Gson.fromJson(Gson.java:990)
        at com.google.gson.Gson.fromJson(Gson.java:929)
        at me.videogamesm12.omegatrack.storage.OTFlags.load(OTFlags.java:32)
        at me.videogamesm12.omegatrack.OmegaTrack.<init>(OmegaTrack.java:26)
        at com.github.hhhzzzsss.epsilonbot.EpsilonBot.<init>(EpsilonBot.java:72)
        at com.github.hhhzzzsss.epsilonbot.Main.<clinit>(Main.java:6)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private java.lang.String java.util.regex.Pattern.pattern accessible: module java.base does not "opens java.util.regex" to unnamed module @5ed828d
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
        at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
        at com.google.gson.internal.reflect.ReflectionHelper.makeAccessible(ReflectionHelper.java:19)
        ... 21 more
```

File: src/main/java/me/videogamesm12/omegatrack/exception/StartFailureException.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package me.videogamesm12.omegatrack.exception;
+
+public class StartFailureException extends Exception {
```
Comment (VideoGameSmash12) [inline src/main/java/me/videogamesm12/omegatrack/exception/StartFailureException.java:3]: allman's style

File: src/main/java/me/videogamesm12/omegatrack/OmegaTrack.java [hunk 2]
```diff
@@ -1,36 +1,46 @@
...
-        try
-        {
-            postgreSQLStorage = new PostgreSQLStorage();
-        }
-        catch (Exception ex)
-        {
-            ex.printStackTrace();
+        this.epsilonBot = epsilonBot;
+        this.flags = OTFlags.load(this.epsilonBot);
+        try {
```
Comment (VideoGameSmash12) [inline src/main/java/me/videogamesm12/omegatrack/OmegaTrack.java:27]: allman's style

File: src/main/java/me/videogamesm12/omegatrack/Tracker.java [hunk 3]
```diff
@@ -2,46 +2,38 @@
...
-            }
-        }, 0, 3000, TimeUnit.MILLISECONDS);
+        this.omegaTrack = omegaTrack;
+        this.timer = this.omegaTrack.timer;
+        this.epsilonBot = this.omegaTrack.epsilonBot;
+        this.trackerTask = new TrackerTask(this.omegaTrack);
+    }
 
-        EpsilonBot.INSTANCE.getSession().addListener(this);
+    public void start() {
```
Comment (VideoGameSmash12) [inline src/main/java/me/videogamesm12/omegatrack/Tracker.java:34]: allman's style

**Comment by allinkdev** [comment]
> Flags do not work properly
> 
> ```
> com.google.gson.JsonIOException: Failed making field 'java.util.regex.Pattern#pattern' accessible; either change its visibility or write a custom TypeAdapter for its declaring type
>         at com.google.gson.internal.reflect.ReflectionHelper.makeAccessible(ReflectionHelper.java:22)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:158)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
>         at com.google.gson.Gson.getAdapter(Gson.java:501)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:116)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:165)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
>         at com.google.gson.Gson.getAdapter(Gson.java:501)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:116)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:165)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
>         at com.google.gson.Gson.getAdapter(Gson.java:501)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.createBoundField(ReflectiveTypeAdapterFactory.java:116)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.getBoundFields(ReflectiveTypeAdapterFactory.java:165)
>         at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory.create(ReflectiveTypeAdapterFactory.java:101)
>         at com.google.gson.Gson.getAdapter(Gson.java:501)
>         at com.google.gson.Gson.fromJson(Gson.java:990)
>         at com.google.gson.Gson.fromJson(Gson.java:929)
>         at me.videogamesm12.omegatrack.storage.OTFlags.load(OTFlags.java:32)
>         at me.videogamesm12.omegatrack.OmegaTrack.<init>(OmegaTrack.java:26)
>         at com.github.hhhzzzsss.epsilonbot.EpsilonBot.<init>(EpsilonBot.java:72)
>         at com.github.hhhzzzsss.epsilonbot.Main.<clinit>(Main.java:6)
> Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private java.lang.String java.util.regex.Pattern.pattern accessible: module java.base does not "opens java.util.regex" to unnamed module @5ed828d
>         at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
>         at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
>         at java.base/java.lang.reflect.Field.checkCanSetAccessible(Field.java:178)
>         at java.base/java.lang.reflect.Field.setAccessible(Field.java:172)
>         at com.google.gson.internal.reflect.ReflectionHelper.makeAccessible(ReflectionHelper.java:19)
>         ... 21 more
> ```

Fixed in https://github.com/VideoGameSmash12/OmegaTrack/pull/2/commits/abe25fa0e247f4ff58c198d061fda43ac14c164c

**Review by VideoGameSmash12 — CHANGES_REQUESTED** [review]
formatting needs to be fixed

**Review by VideoGameSmash12 — APPROVED** [review]
If this doesn't work I'll revert it but it looks good to me

**Pull request merged**